test/pools/pool_os_alloc.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2024 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "pool.hpp"
+#include "poolFixtures.hpp"
+
+INSTANTIATE_TEST_SUITE_P(PoolOsAlloc, umfPoolTest,
```

Comment:
What do you want to test here? We already have a test that uses MALLOC_POOL_OPS (memoryPoolAPI.cpp:143).

Reply:
Right. Replaced with `memoryPoolAPI.cpp`. 
`pool_os_alloc.cpp` removed.
Done.